fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/impl/v1/datagen/DatagenEntrypoint.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.impl.v1.datagen;
```

Comment:
Should the datagen code be put on a different source set?

Reply:
I don't think it matters as it is never run outside of datagen

Reply:
Hmm, do we want the generated JSONs to be committed to the repo or be run automatically each time `build` is run? I think the latter requires the datagen source to be in a different set as it can't depend on its own output.

Reply:
Moved to its own srcset